config/packages/fos_elastica.yaml

```diff
@@ -134,6 +134,11 @@ fos_elastica:
                             properties:
                                 doi:
                                     type: 'text'
+                        projectDirectors:
```

Comment:
Why is this diff showing here? This was done in another PR.

Reply:
probably just a github artifact. I didn't touch this file. I feel it's best to compare outside of github's gui at this point.